easybuild/framework/easyblock.py

```diff
@@ -2859,6 +2860,80 @@ def init_ext_instances(self):
             pbar_label += "(%d/%d done)" % (idx + 1, exts_cnt)
             self.update_exts_progress_bar(pbar_label)
 
+    def update_exts_list_versions(self):
+        """
+        Update all extensions in exts_list to the latest version.
+        """
+
+        # aesthetic print
+        print()
+
+        # loop over all extensions and update their version
+        for ext in self.exts:
+
+            # get metadata of the latest version of the extension
+            metadata = get_pkg_metadata(pkg_class=self.cfg.get('exts_defaultclass', None),
+                                        pkg_name=ext.get('name', None))
+
+            # process the metadata
+            if metadata:
+
+                # build the package
+                pkg = {"name": metadata['name'],
+                       "version": metadata['version'],
+                       "options": {"checksums": [metadata['checksum']]}}
+
+                # clean the package metadata values
+                clean_pkg_metadata(pkg)
```

Comment:
This code is a bit akward, but it will be usefull later with new feature complete_exts_list